client/my-sites/domains/domain-management/settings/index.tsx

```diff
@@ -208,8 +208,8 @@ const Settings = ( {
 		}
 
 		return areAllWpcomNameServers()
-			? translate( 'Your domain is pointing to WordPress.com', { textOnly: true } )
-			: translate( 'Your domain is pointing to custom name servers', { textOnly: true } );
+			? translate( 'Your domain is using WordPress.com name servers', { textOnly: true } )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Domain not using WordPress.com name servers' )` *ES Score: 6*